src/components/Activity.js

```diff
@@ -65,6 +65,8 @@ export default class Activity extends Component {
 	if (props.participant_counter != null)
 		participants = ` (${props.participant_counter})`;
 
+    console.log(props.active);
```

Comment:
I guess this is in here for development purposes only?

Reply:
Yes, forgot to remove that